bin/vitamin.js

```diff
@@ -147,8 +146,8 @@ const build = (options, hotCallback, restartServer) => (options.hot ?
 
 
 const test = (args) => {
-    npmRunSpawn(
-        'jest',
+    spawn(
+        vitaminResolve('node_modules', '.bin', 'jest'),
```

Comment:
Mmmmh, it's not guaranteed to work, and it should not work if jest is not installed inside vitaminJS.
It's better to use just spawn(`yarn jest`), and then, add 
'--' before the list of arguments.